bin/tei_items_to_html.xsl

```diff
@@ -39,80 +37,209 @@
-                        <a class="page-link" href="/partners.html">Partners</a>
-                        <a class="page-link" href="/research.html">Research</a>
-                        <a class="page-link" href="/contributor.html">Contribute</a>
-                        <a class="page-link" href="/contact.html">Contact</a>
-                        <a class="page-link" href="/news.html">News</a>
+                        <a class="page-link" href="/catalogo">Catalogo</a>
                     </div>
                 </nav>
             </div>
         </header>
     </xsl:template>
-    
+
     <xsl:template match="tei:teiCorpus">
         <xsl:variable name="ciconum">
-            <xsl:value-of select="tei:teiHeader/tei:fileDesc/tei:publicationStmt/tei:idno[@type='cico']"/>
+            <xsl:value-of
+                select="tei:teiHeader/tei:fileDesc/tei:publicationStmt/tei:idno[@type = 'cico']"/>
         </xsl:variable>
         <xsl:result-document href="{$ciconum}.html" indent="yes">
-        <html xmlns="http://www.w3.org/1999/xhtml" lang="it">
-            <head>
-                <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"></meta>
-                <title>
-                    <xsl:value-of select="tei:teiHeader/tei:fileDesc/tei:titleStmt/tei:title" />
-                </title>
-                <link rel="stylesheet" href="{$css}"/>
-            </head>
-            
-            <body>
-                <xsl:call-template name="createSiteMastHead"/>
-
-                <nav>
-                  <ul class="breadcrumb">
-                    <li>
-                      <a href="../index.html" class="browse-catalogo-button">Catalogo</a>
-                    </li>          
-                  </ul>
-                </nav>
-                
-                <header id="metadata">
-                  <div class="bibl">
-                    <xsl:apply-templates select="tei:teiHeader/tei:fileDesc/tei:sourceDesc/tei:bibl" />
-                  </div>
-                </header>
-                
-                <main id="main">
-                    <iframe allowfullscreen="true" src="{$viewpages}/{$ciconum}.html" />
-                </main>
-                <xsl:call-template name="createSiteFooter"/>
-            </body>
-        </html>
+            <html xmlns="http://www.w3.org/1999/xhtml" lang="it">
+                <head>
+                    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+                    <title>
+                        <xsl:value-of select="tei:teiHeader/tei:fileDesc/tei:titleStmt/tei:title"/>
+                    </title>
+                    <link rel="stylesheet" href="{$css}"/>
+                </head>
+
+                <body>
+                    <xsl:call-template name="createSiteMastHead"/>
+
+                    <main id="main">
+                        <h1>
+                            <xsl:value-of select="concat('Catalogo Cicognara item ', $ciconum)"/>
```

Comment:
I think it would be better to keep the existing display (the Catalogo text) than using the generic "Catalogo Cicognara item #".

Reply:
fixed in latest PR